cmake/add_library_macros.cmake

```diff
@@ -235,6 +235,18 @@ function(openms_add_library)
     set(has_dll_dep
             $<BOOL:$<TARGET_RUNTIME_DLLS:${openms_add_library_TARGET_NAME}>>
             )
+ ## QuickWidgets is a runtime-only dependency that we need to copy and install when WebEngine is found.
```

Comment:
If you put it inside the install_library macro it will call find_package everytime that macro is called.
That's why we actually had it outside. 

Also note that the macro is used for every library but only OpenMS_GUI depends on WebEngine (optionally). So, looking for it in OpenSwath and OpenMS is not ideal.

Reply:
ah, I see. Let me fix that.

Reply:
@jpfeuffer how's it look now?